packages/protocol-kit/tests/e2e/offChainSignatures.test.ts

```diff
@@ -352,7 +324,9 @@ describe('Off-chain signatures', () => {
         trusted: true,
         signatures: '0x111111222222'
       }
-      const signedTx = await safeSdk.signTransaction(safeServiceTransaction)
+      const signedTx = await safeSdk.signTransaction(
+        safeServiceTransaction as SafeMultisigTransactionResponse
```

Comment:
In this one is even more necessary to have the assistance at object creation time instead of casting.

Reply:
yes, changed